addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3010,6 +3010,8 @@ var SnippetsMenu = Widget.extend({
                 break;
         }
 
+        this.options.wysiwyg.toolbar.el.classList.remove('oe-floating');
```

Comment:
How can we end up with that class if we're in snippets bar? Shouldn't it never be floating when there is a snippets bar?

Reply:
That's it. By default it's floating, as the SnippetMenu hack the toolbar, it make sense that it also remove it's floating class.
There is obviously cleaner that might require more refactoring.